internal/workflow/processing_test.go

```diff
@@ -1061,3 +1072,400 @@ func (s *ProcessingWorkflowTestSuite) TestPreprocessingChildWorkflow() {
 	s.True(s.env.IsWorkflowCompleted())
 	s.NoError(s.env.GetWorkflowResult(nil))
 }
+
```

Comment:
I tried to parameterize these tests, just using a for loop and calling `s.SetupWorkflowTest(cfg)` on each loop. I think that may be enough, but it makes it hard to debug the tests individually.

Since the duplication in these workflow tests is already an issue, it may be better that we try to improve it all together in a different PR.

Reply:
Yeah, the processing workflow tests are long and repetitive.